components/monitoring/base/install_prometheus.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-user-workload-monitoring-stack
+  namespace: openshift-user-workload-monitoring
+spec:
+  targetNamespaces:
+  - openshift-user-workload-monitoring
+
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: prometheus
+  namespace: openshift-user-workload-monitoring
+spec:
+  channel: beta
+  installPlanApproval: Automatic
+  name: prometheus
+  source: community-operators
```

Comment:
Don't we want to have supported environment? Community operator from beta channel does not seem to be right to me.

Reply:
Prometheus is available in beta channel from comunity operators only in OSD. I dont see much option there. And I believe we can trust the operators published in openshift operatorhub :thinking:

Reply:
I agree with Bama here - it's the only option I found.

Reply:
Available Prometheus monitors as of now:
<img width="889" alt="image" src="https://user-images.githubusercontent.com/1302432/196158822-87a8c20f-9dc8-457a-9b27-be167f23c83c.png">
The AWS one sends it to their Prometheus service ($$), and the others don't actually deploy Prometheus.